synapse/storage/databases/main/cache.py

```diff
@@ -334,6 +341,14 @@ def _invalidate_state_caches_and_stream(
                 txn, CURRENT_STATE_CACHE_NAME, [room_id]
             )
 
+    async def send_invalidation_to_replication(self, cache_name, keys):
```

Comment:
new methods should be typed :)

```suggestion
    async def send_invalidation_to_replication(self, cache_name: str, keys: Optional[Collection[Any]]) -> None:
```